simln-lib/src/lib.rs

```diff
@@ -519,6 +518,8 @@ pub struct Simulation {
     activity: Vec<ActivityDefinition>,
     /// Results logger that holds the simulation statistics.
     results: Arc<Mutex<PaymentResultLogger>>,
+    /// track all tasks spawned for use in the simulation.
+    tasks: TaskTracker,
```

Comment:
nitnit: capitalize "track" and make a note that the `run` function will wait for all tasks to complete so that people passing a tracker in that they're using elsewhere know that it'll be handled. 

I think that we can put that both here and on `run` because it's important.